library/archlinux

```diff
@@ -6,7 +6,10 @@ Maintainers: Santiago Torres-Arias <santiago@archlinux.org> (@SantiagoTores),
              Pierre Schmitz <pierre@archlinux.de> (@pierres)
 GitRepo: https://github.com/archlinux/archlinux-docker.git
 
+Tags: latest, 20191105
+GitCommit: 15a9a3997aa8e571a8ce2a019b65b5eb698919ec
+GitFetch: refs/tags/20191105
 
-Tags: latest, 20191006
+Tags: 20191006
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme, but the tag will still be available to users who want to `docker pull` it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
From IRC:
> oh, I think we want to keep that around for some time. We're still settling on our policy for old images but for at least the next 6 or so we'll keep around